lib/liquid_swipe.dart

```diff
@@ -203,8 +194,8 @@ class _LiquidSwipe extends State<LiquidSwipe> with TickerProviderStateMixin {
 
   next() {
     _LiquidSwipe().setState(() {
-      activePageIndex += 1;
-      nextPageIndex = activePageIndex + 1;
+      activePageIndex = _activePageIndex + 1;
```

Comment:
i am talking about this, activePageIndex is not valid? isn't it?

Reply:
`activePageIndex = _activePageIndex + 1;`

is just a call of the setter function:

```
set activePageIndex(int value) {
    _activePageIndex = value;

    widget.onPageChangeCallback(_activePageIndex);
  }
```